doc/api/child_process.md

```diff
@@ -1061,10 +1061,12 @@ added: v0.1.90
 -->
 
 * `signal` {number|string}
+* Returns: {boolean}
 
 The `subprocess.kill()` method sends a signal to the child process. If no
 argument is given, the process will be sent the `'SIGTERM'` signal. See
-signal(7) for a list of available signals.
+signal(7) for a list of available signals. This function returns `true` on
+successful signal delivery, and `false` otherwise.
```

Comment:
Nitpickiest of nitpickiest but if "signal delivery" is understood to mean "signal received by the target process" then this statement is false because the target process might have blocked it off.

kill(2) will still indicate success even though it may have silently discarded or coalesced the signal if one was already pending.

There are also some interesting edge cases around e.g. SIGSTP or SIGTTOU causing already pending SIGCONT signals to get discarded. I.e., you can't be sure delivery will actually take place.

Reply:
@bnoordhuis what about this instead:

> This function returns `true` if `kill(2)` succeeds, and `false` otherwise.

Reply:
Love it, that's precise w/o being too involved.

Reply:
@cjihrig Yes, sounds good.